base/applications/regedit/regedit.c

```diff
@@ -158,8 +158,9 @@ BOOL PerformRegAction(REGEDIT_ACTION action, LPWSTR s, BOOL silent)
                     int choice;
 
                     LoadStringW(hInst, IDS_IMPORT_PROMPT, szText, COUNT_OF(szText));
+                    SetClassLongPtrW(NULL, GCL_STYLE, GetClassLongPtrW(NULL, GCL_STYLE) | CS_NOCLOSE);
```

Comment:
I have no idea how did you get a message box with a disabled close button using this trick :zany_face: Because `InfoMessageBox` function is specific to regedit, and it internally calls `MessageBoxW`, whose window style you cannot control. :thinking:

Reply:
And it doesn't work for me when I apply the patch locally, as I expected. There are some hacky solutions available though: https://stackoverflow.com/questions/13189791/disable-x-button-icon-on-the-top-of-the-right-in-messagebox-using-c-win32-api (but I probably gonna just revert this)